openmdao/drivers/tests/test_scipy_optimizer.py

```diff
@@ -152,6 +155,48 @@ def test_opt_distcomp(self):
                           1e-5)
 
 
+@unittest.skipUnless(MPI, "MPI is required.")
```

Comment:
Should add a check for PETSc here as well since the MPI stuff won't work without PETSc (we use PETSc to do our scatters between Vectors under mpi)